babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
why do we need this change?

Reply:
I see it was removed but I'm still curious why it was added. Do we need this? What did it solve?